charts/sourcegraph/examples/external-redis/README.md

```diff
@@ -0,0 +1,53 @@
+# Using external Redis instances
+
+Sourcegraph deployment by default ships two separate Redis instances for different purposes
+
+- [redis-cache.Deployment.yaml](../../templates/redis/redis-cache.Deployment.yaml)
+- [redis-store.Deployment.yaml](../../templates/redis/redis-store.Deployment.yaml)
+
+When using external Redis instances, you’ll need specify the corresponding environment variable for [each of the following deployments](https://docs.sourcegraph.com/admin/install/kubernetes/configure#configure-custom-redis)
+
+## Optional 1 - One shared external Redis instance
+
+Example values override [override-shared.yaml](./override-shared.yaml).
+
+### `REDIS_ENDPOINT`
+
+The string must either have the format `$HOST:PORT` or follow the [IANA specification for Redis URLs](https://www.iana.org/assignments/uri-schemes/prov/redis) (e.g., redis://:mypassword@host:6379/2)
+
+## Option 2 - Two separate external Redis instances
+
+Example values override [override-separate.yaml](./override-separate.yaml).
+
+### `REDIS_CACHE_ENDPOINT`
+
+The string must either have the format `$HOST:PORT` or follow the [IANA specification for Redis URLs](https://www.iana.org/assignments/uri-schemes/prov/redis) (e.g., redis://:mypassword@host:6379/2)
+
+### `REDIS_STORE_ENDPOINT`
+
+The string must either have the format `$HOST:PORT` or follow the [IANA specification for Redis URLs](https://www.iana.org/assignments/uri-schemes/prov/redis) (e.g., redis://:mypassword@host:6379/2)
+
+## Notes
+
+You may store these sensitive environment variables in a [Secret](https://kubernetes.io/docs/concepts/configuration/secret/).
```

Comment:
🤔  Are redis endpoints considered sensitive info? Doesn't seem like it matters much - if the endpoint is routable you're already in a bad state. Maybe better to show as a ConfigMap example?

Reply:
> 🤔 Are redis endpoints considered sensitive info? Doesn't seem like it matters much - if the endpoint is routable you're already in a bad state. Maybe better to show as a ConfigMap example?

The `ENDPOINT` env var may or may not include credential `redis://somepassword@external-redis/0`, so it could be sensitive.

But good point, we can add a ConfigMap example.